### PR TITLE
Added some debian dependencies for the new QML skin

### PR DESCRIPTION
**Pull request by daschuer**
Unfortunatly the QMLDemo skin does still not load on Ubuntu Groovy (Qt 5.12): 

```
info [Main] Loaded skin "QMLDemo"
warning [Main] file:///home/sperry/workspace/mixxx/res/skins/QMLDemo/main.qml:54:9: Type Skin.DeckRow unavailable 
             Skin.DeckRow { 
             ^
warning [Main] file:///home/sperry/workspace/mixxx/res/skins/QMLDemo/DeckRow.qml:71:5: Type Deck unavailable 
         Deck { 
         ^
warning [Main] file:///home/sperry/workspace/mixxx/res/skins/QMLDemo/Deck.qml:25:5: Type Skin.ControlSlider unavailable 
         Skin.ControlSlider { 
         ^
warning [Main] file:///home/sperry/workspace/mixxx/res/skins/QMLDemo/ControlSlider.qml:5:1: Type Skin.Slider unavailable 
     Skin.Slider { 
     ^
warning [Main] file:///home/sperry/workspace/mixxx/res/skins/QMLDemo/Slider.qml:7:1: Type MixxxControls.Slider unavailable 
     MixxxControls.Slider { 
     ^
warning [Main] file:///home/sperry/workspace/mixxx/res/skins/QMLDemo/Mixxx/Controls/Slider.qml:9:31: Invalid alias target location: margins 
         property alias barMargin: barShape.anchors.margins 
                                   ^
warning [Main] Skin "QMLDemo" failed to load!
```
Is something else missing?

### CODE REVIEW

File: tools/debian_buildenv.sh [hunk 1]
```diff
@@ -84,5 +84,9 @@ case "$COMMAND" in
             protobuf-compiler \
             qt5keychain-dev \
             qtdeclarative5-dev \
+            qml-module-qtquick-controls \
```
Comment (uklotzde) [inline tools/debian_buildenv.sh:87]: On Fedora it's just qt5-qtquickcontrols + qt5-qtquickcontrols2.

**Comment by Holzhaus** [comment]
Apparently aliasing associated properties in QML has been added in Qt >= 5.12.


I tried to write a "smoke test" for QML skins, but since skins require COs, this basically forces me to reimplement the whole MixxxMainWindow constructor in a test. We should definitely split QWisgets/UI stuff off to make that usable in a test.

**Comment by Holzhaus** [comment]
> I tried to write a "smoke test" for QML skins, but since skins require COs, this basically forces me to reimplement the whole MixxxMainWindow constructor in a test. We should definitely split QWisgets/UI stuff off to make that usable in a test.

I give up. 



If someone else wants to take a stab as this: https://github.com/mixxxdj/mixxx/compare/main...Holzhaus:qmlskintest?expand=1

File: tools/debian_buildenv.sh (hunk 1 above)
Comment (Holzhaus) [reply tools/debian_buildenv.sh:87]: Weird, I didn't install any additional dependencies on Arch. Just qt5-declarative.

Comment (uklotzde) [reply tools/debian_buildenv.sh:87]: Both qt5-qtquickcontrols and qt5-qtquickcontrols2 are installed on my system. Maybe they are not needed? I need to check this with a mock build sometime.

Comment (uklotzde) [reply tools/debian_buildenv.sh:87]: At least they are not required for qt5-declarative. If we need them we have to install them explicitly.

**Comment by daschuer** [comment]
Currently I cannot test the new QML skin because of the debug message above. 
It there a way to make it work for example by comment out the parts that requires qt 5.14?

By the way this PR was created by installing the packages Mixxx was prompting me when rejecting the QMLDeno and fall back to default.

**Comment by Swiftb0y** [comment]
@daschuer the issue you are experiencing should be fixed by #3963

**Pull request merged**